src/pages/dev/axelarjs-sdk/tx-status-query-recovery.mdx

```diff
@@ -25,38 +37,55 @@ const sdk = new AxelarGMPRecoveryAPI({
 });
 ```
 
-### Query transaction status by txHash
+## Query transaction status by `txHash`
 
-Invoke `queryTransactionStatus`:
+See the status of a transaction by passing its `txHash` into [`queryTransactionStatus()`](https://github.com/axelarnetwork/axelarjs-sdk/blob/86bc55224f84f9539dd109e5dfd6b443a116eebe/src/libs/TransactionRecoveryApi/AxelarRecoveryApi.ts#L226):
 
 ```ts
 const txHash: string =
   "0xfb6fb85f11496ef58b088116cb611497e87e9c72ff0c9333aa21491e4cdd397a";
-const txStatus: GMPStatusResponse = await sdk.queryTransactionStatus(txHash);
+const eventIndex: number = 97
+const txStatus: GMPStatusResponse = await sdk.queryTransactionStatus(txHash, eventIndex);
```

Comment:
querying with `eventIndex` can be useful but is not required. For example it's great if there are a number of internal txs but if there is just a single tx for that tx hash then it could be easier to just use the hash and ignore the `eventIndex`. Could be worth being explicit about that